tools/pot/openvino/tools/__init__.py

```diff
@@ -1,3 +1,10 @@
-# Copyright (C) 2020-2022 Intel Corporation
+# Copyright (C) 2018-2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
-__path__ = __import__('pkgutil').extend_path(__path__, __name__)
+
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)
+
+try:
+    import openvino.tools.mo
+except ImportError:
+    import warnings
+    warnings.warn("openvino.tools.mo module could not be found!", ImportWarning, stacklevel=2)
```

Comment:
do we need to change this file at all? 
I suppose a default file with `pkgutil` is enough, is not it?